cardano-submit-api/cardano-submit-api.cabal

```diff
@@ -52,6 +52,7 @@ library
                       , optparse-applicative-fork
                       , ouroboros-consensus-cardano
                       , ouroboros-network
+                      , ouroboros-network-protocols
```

Comment:
either add a bound on `cardano-api` or bounds on network packages.  The same in other `cabal` files.

Reply:
This is a problem that inflicts lots of packages.  I'm intending to add them in a separate PR and fix it up for all packages.